### PR TITLE
Dockerize

### PR DESCRIPTION
**Pull request by jgrund**
Update iml-update-check to be a separate docker service.

  - Add Dockerfile.
  - Add script that will wait until manager settings are available in
the container.

Signed-off-by: Joe Grund <jgrund@whamcloud.io>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM node:alpine
+WORKDIR /usr/src/app
+COPY handler.js .
+EXPOSE 8080
```
Comment (johnsonw) [inline Dockerfile]: Any reason the copy commands aren't together?

Comment (jgrund) [reply Dockerfile]: Any reason they need to be?

Comment (johnsonw) [reply Dockerfile]: From a readability standpoint it seems cleaner but other than that, no.

Comment (jgrund) [reply Dockerfile]: The reason I have them grouped like this is so it's clear that the wait-for-settings.sh is separate from what this container does.

By grouping them, it makes this less clear and means we need to hunt around for the parts when we want to remove the wait-for functionality.

Comment (johnsonw) [reply Dockerfile]: fair enough 👍

**Pull request merged**